ci-operator/config/openshift/builder/openshift-builder-master.yaml

```diff
@@ -3,18 +3,26 @@ base_images:
     name: "4.6"
     namespace: ocp
     tag: base
+  ocp_builder_golang-1.13:
+    name: builder
+    namespace: ocp
+    tag: rhel-8-golang-openshift-4.6
   openshift_release_golang-1.13:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: rhel-8-release-openshift-4.6
 build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: rhel-8-release-openshift-4.6
 images:
-- from: base
+- dockerfile_path: Dockerfile.rhel8
+  from: base
```

Comment:
from: base-8 ??

https://github.com/openshift/release/blob/master/ci-operator/config/openshift/images/openshift-images-master.yaml#L28-L31

Reply:
@yselkowitz so perhaps this PR was misleading in its tile. There will be a follow-up PR that makes the ubi-8 switch real.